addons/resource.language.en_gb/resources/strings.po

```diff
@@ -8887,7 +8887,13 @@ msgctxt "#15315"
 msgid "Save progress to a new save file"
 msgstr ""
 
-#empty strings from id 15316 to 15999
+#. Label for autosave
```

Comment:
Can you give a more thorough description of the string? You can never be too verbose to assist foreign language translators understand the string's purpose!

Here's something you can use:

```
Label for a savestate in a video game which was automatically created without user input.
```

IMO any bit extra helps.